app/enterprise/0.34-x/property-reference.md

```diff
@@ -2,27 +2,63 @@
 title: Configuration Property Reference for Kong Enterprise
 ---
 
-## Kong Manager
+## General
```

Comment:
Could we include something here to explain why the first section is just links to another page? It does seem odd to just say "see Kong's configuration reference" as users might be like, "I thought I was!' IDK something like  - 'properties in this section are in both Kong and Kong Enterprise. Please see the Kong configuration file for further information"